scripts/image_functions.sh

```diff
@@ -38,7 +39,7 @@ function image_test() {
 
 function image_deploy() {
         # set older versions non-public
-        images=($(glance image-list --owner "$OS_USERNAME" --property name="$IMAGE_NAME" \
+        images=($(glance image-list --property owner="$OS_TENANT_ID" --property name="$IMAGE_NAME" \
```

Comment:
I was a bit weird out by the --owner -> --property owen="" but this makes it look more like the openstack command.